draft-ietf-quic-transport.md

```diff
@@ -1829,9 +1829,7 @@ The fields in the MAX_DATA frame are as follows:
 Maximum Data:
 
 : A 64-bit unsigned integer indicating the maximum amount of data that can be
-  sent on the entire connection, in units of 1024 octets.  That is, the updated
-  connection-level data limit is determined by multiplying the encoded value by
-  1024.
+  sent on the entire connection, in units octets.
```

Comment:
of ?

Reply:
imagine how many mistakes I can make in a diff more than 1 line long.